FluentUI.Demo/src/main/assets/dogfood-release-notes.txt

```diff
@@ -1,10 +1,12 @@
-Release Version 0.1.33:
+Release Version 0.1.34:
 
 What's New:
 
 Fluent V2:
 	Fix/Enhancement:
-		1. Added Functionality to the BottomSheet with TestCases: Enable/Disable Swipe Down Dismiss with more intuitive accessibility controls
-		2. ToolTip Alignment Bug Fixed: Resolved an issue where ToolTips were not aligning correctly, ensuring a visually consistent and user-friendly interface
-		3. CardNudge Enhancement: Fixed a right-to-left (RTL) issue affecting swiping left and right within CardNudge, ensuring a smoother interaction
-		4. Vertical Lists Icon Color Consistency: Addressed an issue where disabled icons in vertical lists had inconsistent colors. Icons now maintain uniform appearance for improved visual clarity
+	    1. Persona chip Bug Fixed: Persona height will change when the system font size is increased
+		2. Improved the visibility of the ToolTip Activity Caret in dark mode
```

Comment:
Can you check the formatting

Reply:
![image](https://github.com/microsoft/fluentui-android/assets/68989156/17d0d983-c6a6-4ea1-a9e2-e77dc4e4ca68)
Seems okay in local IDE